### PR TITLE
Statsreceiver config file update (webproxy, healthcheck, orbiter)

### PR DESCRIPTION
**Pull request by grafael**
Update the config file to match what is currently running on statsreceiver VM

### CODE REVIEW

File: statreceiver.lua [hunk 1]
```diff
@@ -33,10 +33,16 @@ influx_base = "http://influx-internal.datasci.storj.io:8086"
 influx_user = os.getenv("INFLUX_USERNAME")
 influx_pass = os.getenv("INFLUX_PASSWORD")
 
+influx_wp_base = "http://influx-wp.datasci.storj.io:8086"
+influx_wp_user = os.getenv("INFLUX_WP_USERNAME")
+influx_wp_pass = os.getenv("INFLUX_WP_PASSWORD")
+
 v3_url = string.format("%s/write?db=v3_stats_new&u=%s&p=%s",influx_base, influx_user, influx_pass)
+wp_url = string.format("%s/write?db=v3_stats_new&u=%s&p=%s",influx_wp_base, influx_wp_user, influx_wp_pass)
```
Comment (stefanbenten) [inline statreceiver.lua:41]: NIT: space missing after the first comma.

**Review by stefanbenten — DISMISSED** [review]
Just a few NITs

File: statreceiver.lua [hunk 2]
```diff
@@ -99,3 +110,4 @@ destination = pbufprep(pcopy(
 deliver(source, destination)
 
 
+
```
[the remark this answers is not in the document]
Comment (grafael) [reply statreceiver.lua]: Fixed

File: Jenkinsfile [hunk 3]
```diff
@@ -74,4 +75,4 @@ pipeline {
             deleteDir()
         }
     }
-}
+}
```
[the remark this answers is not in the document]
Comment (grafael) [reply Jenkinsfile]: Fixed

**Review by ethanadams — DISMISSED** [review]

**Review by stefanbenten — APPROVED** [review]

File: go.mod [hunk 4]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Shopify/go-lua v0.0.0-20191113154418-05ce435a9edd
 	github.com/jtolds/go-luar v0.0.0-20200310225017-6fa637b8208b
 	github.com/lib/pq v1.3.0
-	github.com/mattn/go-sqlite3 v2.0.3+incompatible
+	github.com/mattn/go-sqlite3 v1.14.12 // indirect
```
Comment (stefanbenten) [inline go.mod:9]: Isnt this a downgrade?

Comment (stefanbenten) [reply go.mod:9]: Nvm, storj/storj looks similar: https://github.com/storj/storj/blob/main/go.mod

**Review by stefanbenten — APPROVED** [review]

File: go.mod (hunk 4 above)
Comment (grafael) [reply go.mod:9]: No, see: https://github.com/mattn/go-sqlite3

**Review by ethanadams — APPROVED** [review]

**Pull request merged**